reducers/AddRepo.js

```diff
@@ -0,0 +1,16 @@
+const AddRepo = (state = [], action) => {
```

Comment:
Reducer's should be `camelCase` too. Also, I don't get this naming decision. Why `AddRepo`, isn't it the action which actually populate the state? Something like `repos` makes more sense, IMO.